### DISCUSSION TITLE
CEBRA Neuroimaging fMRI

### DISCUSSION THREAD

**Discussion by navidivan** [AdaptiveMotorControlLab/CEBRA#154] [General]
Just wondering if anyone has used CEBRA dimensionality reduction for neuroimaging data. My search does not show anything on the web. Was wondering whether it has been try or why this hasn't been tried yet,

**Comment by MMathisLab** [comment]
They have :). Here is a paper from last year, which @timonmerk lead: https://www.researchsquare.com/article/rs-3212709/v1
